### ISSUE TITLE
Does the SDK supported multiple destinations or multiple cc or multiple bcc?

### ISSUE THREAD

**Issue by issacjisen** [infobip/infobip-api-java-client#20]
Dear team,      
      Our team wants to send email or SMS via infobip platform. But I haven't found the function which is using for multiple destinations/cc/bcc in the SDK (3.2.0). 
       If the SDK doesn't support it, should I send a request for each recipient? Or can I set each recipient in the request body? such as the fllow screenshot,
       Your reply will be really appreciated.

![image](https://user-images.githubusercontent.com/93071955/196345127-8b6fd414-59ae-478b-ae56-5709209b2e0f.png)

**Comment by ib-fsrnec** [comment]
Hi @issacjisen, sorry for the late response.

Thank you very much for raising an issue. Yes, it is possible to add multiple recipients through to/cc/bcc form parameters like this:
```
curl --request POST 'https://api.infobip.com/email/3/send' \
...
--form 'to="john.smith@somedomain.com"' \
--form 'to="john.doe@somedomain.com"' \
```

We are going to update our API docs to emphasize this use case. Unfortunately, the SDK itself doesn't provide support for adding multiple recipients. This is going to be included in our new major release that should be live in a few weeks. We'll keep you informed.

Thank you for using our library!
Best regards,
Filip

**Comment by issacjisen** [comment]
Hi @ib-fsrnec ,

Thank your for replying. We have already use your API to implement the multiple recipients. 

By the way, I want to ask a new question. If the email has hyperlink of content, I saw that the actually link of the email which i received has changed to another domain. When I click it, it can truns to the orign link. 

May I know how we can avoid this situation? I have sent email to infobip@infobip.com & selfserve@infobip.com. But I didn't receive the reply. 

Could you give me some solution for this situation?

![image](https://user-images.githubusercontent.com/93071955/212645727-ecd48ab8-2127-43e4-955a-df79270dd47b.png)
![image](https://user-images.githubusercontent.com/93071955/212645776-76c49074-5c5b-4087-8a9c-17b470af62ee.png)
The hyper link was:
https://tracking.selfserviceib.com/tracking/1/click/ejoRCcw7cfKwhilpbcPgu0JLmTxNR5wRW-8-gE0yPJO92tYye8tt8W-vMbWMSv-iueqqoXeTnz1AOlMRRbe0qbGhvtzXdMVN64_7DLn-pOY4JyNnYgHSsZIPYP_ZqVi-Rv04dxhcM8vdZe7akA6zJ6GaSDrL9h1oKmwM4FUCj2mVt40vIsawK0ZZKVZZ-4rF5Kr-7tJJdK5l1UvVki0TJU5KdyVmMQJKfDiiDSyaTpd6bJvkZXWE8nnQs_rQ0CWd8hA3aopIHp7cqBF3q3MInEwJxPzsa19W3ZjkL5-x7sY84AWuilKyaXHJbTBI1dpsuyNm1_77u6XCH70StQ8pxtlMIX-2D8LNHbXf9pxtG9J2__5SeYID_KwlMzrU7sW1ymzj5lYCtctNtyHd6Z3VjbRkHp6BEfbSpNwlbFc8YbnLTMQLmllPAST81Z509zkILqaSIpQG0947zeD-AJrrpBbg80BWaCsrFlnsLkI7z1ZwRj263H-sNz8AyBqV-jeFsaRA5JSZ3bAkZjLSB_4ldt4q6fY823oqQuPJSFDdOMf7uHZ_W6cq2GrE-yNIG1DrTwW-uCk_RHCL_5E6utoSwtUftVgQ0nqvaM-0ZkK0f5uQKQOHlqnA8XW0cofz4ZkODAfn4df_f0LTx4CXr6GgvbX5svK1aJl6djD6UI8Jh0btkmOZFY3tHDbJrOlLtDH8wT2x-f7LzETr6mA_HPJ8nwtk0uJNyZZvsbrO-fBNFDvan5bRi7p37zunwxNxfzj0LiHeh07f9OdQKeDHvYYWONrDn3QI3zKYAHAlfyHHc-swYSpoGWCoUsHOrv69Y5hp7mEf6mYqxabLNkFno0wPAeoCCUG0FhhkAQuvfLwwGP1WfU5O7S0ndg9Arqf8pzi_LNgLEOZPF5ow-QcFwfPCaw==

Please advise if there is any mechanism from Infobip end to automatically modify the hyper link and if this could be stopped.
We would expect to have the link remain unchanged after e-mail sent out.

The bulkId of this Email was: dhf82sacueecbbl1nl4e and the messageId was j86k57ymn3mooeq31dan

If you don't mind, could we cmmunicate via email? My email of company account is jisen.yang@zatech.com.

Or could you leave your email here? I will transmit the email which I have sent to  infobip@infobip.com & selfserve@infobip.com.

Thank you in advance. 

Jason Yang.

**Comment by ib-fsrnec** [comment]
Hi @issacjisen and thank you for your question. Of course we can communicate via email. I was about to suggest the same since your question is not related to the library itself. Feel free to contact us at support@infobip.com. In the meantime, feel free to check our Email product documentation: https://www.infobip.com/docs/email.

**Comment by ib-fsrnec** [comment]
Hi @issacjisen, we released a new major version 4.0.0 yesterday.

Now, it's possible to send an email to multiple destinations using the library code.
```
    EmailSendResponse response = sendEmailApi
                .sendEmail(List.of(givenDestination, givenAnotherDestination))
                .cc(List.of(givenCCDestination, givenAnotherCCDestination))
                .bcc(List.of(givenBCCDestination, givenAnotherBCCDestination))
                .from(givenFrom)
                .subject(givenSubject)
                .replyTo(givenReplyTo)
                .text(givenText)
                .attachment(List.of(givenAttachment))
                .execute();
```

I'm going to close the issue. Feel free to reopen it if you find additional issues.

Best regards,
Filip Srnec
